integration_tests/network.py

```
@@ -151,3 +152,82 @@ def connect_custom_mantra():
+    with (path / "validator.log").open("w") as logfile:
+        cmd = [
+            "start-validator",
+            path,
+        ]
+        print(*cmd)
+        proc = subprocess.Popen(
+            cmd,
+            preexec_fn=os.setsid,
+            stdout=logfile,
+            stderr=subprocess.STDOUT,
+        )
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)